examples/with-viem-and-passkeys/src/pages/index.tsx

```diff
@@ -114,7 +114,7 @@ export default function Home() {
     });
 
     const viemClient = createWalletClient({
-      account: viemAccount,
+      account: viemAccount as Account,
```

Comment:
note to self: do this in our separate repo https://github.com/tkhq/demo-viem-passkeys once this is version bump is released